Software/setup.sh

```diff
@@ -25,17 +17,20 @@ sudo apt-get --assume-yes install python3-setuptools
 
 sudo easy_install3 pip
```

Comment:
I'm honestly not sure why we I added sudo easy_install3 pip. Maybe I needed that to run the program back then. I could be wrong, only way to test is to reset the pi and run the setup then run the program